R/lazyframe__lazy.R

```diff
@@ -1038,14 +1043,15 @@ LazyFrame_unique = function(
 
 #' Group a LazyFrame
 #' @description This doesn't modify the data but only stores information about
-#' the group structure. This structure can then be used by several functions
-#' (`$agg()`, `$filter()`, etc.).
+#'   the group structure. This structure can then be used by several functions
+#'   (`$agg()`, `$filter()`, etc.).
```

Comment:
Why this change?

Reply:
That's the autoformatting that wraps lines so that they don't go over 80 chars (ctrl+shift+/). Apparently it also adds indents. I can remove that